src/core/sys/linux/fcntl.d

```diff
@@ -10,7 +10,10 @@ nothrow:
 enum {
     FALLOC_FL_KEEP_SIZE = 0x01,
     FALLOC_FL_PUNCH_HOLE = 0x02,
-    FALLOC_FL_NO_HIDE_STALE = 0x04
+    FALLOC_FL_NO_HIDE_STALE = 0x04,
+    FALLOC_FL_COLLAPSE_RANGE = 0x08,
+    FALLOC_FL_ZERO_RANGE = 0x10,
+    FALLOC_FL_INSERT_RANGE = 0x20
```

Comment:
Looking at https://github.com/torvalds/linux/blob/v4.11/include/uapi/linux/falloc.h, it seems that 

```
FALLOC_FL_UNSHARE_RANGE		0x40
```

has been added in the meantime.
It was added in October 2016: https://github.com/torvalds/linux/commit/71be6b4942dd64bc17728f82f787be98fd8afed7